javatests/arcs/android/systemhealth/testapp/TestActivity.kt

```diff
@@ -211,6 +216,52 @@ class TestActivity : AppCompatActivity() {
             }
         )
 
+        val serviceProbabilityLabel =
+            findViewById<TextView>(R.id.service_crash_rate_label)
+        findViewById<SeekBar>(R.id.service_crash_rate).also {
+            serviceProbabilityLabel.text =
+                getString(R.string.storage_service_crash_rate, it.progress)
+        }.also {
+            it.setOnSeekBarChangeListener(
+                object : SeekBar.OnSeekBarChangeListener {
+                    override fun onProgressChanged(seekBar: SeekBar?,
+                                                   progress: Int,
+                                                   fromUser: Boolean) {
+                        storageServiceCrashRate = progress
```

Comment:
Do we need a range check [0, 100] here?

Reply:
The document says this value's default range is 0 to 100.